scripts/ci/kubernetes/docker/airflow-init.sh

```diff
@@ -17,9 +17,10 @@
 #  specific language governing permissions and limitations      *
 #  under the License.
 
-cd /usr/local/lib/python2.7/dist-packages/airflow && \
-cp -R example_dags/* /root/airflow/dags/ && \
+set -e
+
+cd /usr/local/lib/python3.7/site-packages/airflow/ && \
+cp -R example_dags/* /home/airflow/dags/ && \
 airflow initdb && \
 alembic upgrade heads && \
-(airflow create_user -u airflow -l airflow -f jon -e airflow@apache.org -r Admin -p airflow || true) && \
-echo "retrieved from mount" > /root/test_volume/test.txt
```

Comment:
Why did you remove this line? Isn't this necessary for the volume mounting tests?

Reply:
I haven't found this necessary in our deployments.

Reply:
So the reason this line didn't break the CI is because a previous PR silently broke all kubernetes tests. I'm actively trying to fix this and will report back when I have it working.